<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Отчет по приходам</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: linear-gradient(to top, #fefcea, #f1da36);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Ткань</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="le_material_provider">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton_2">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton_11">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(170, 0, 0);
color: rgb(255, 255, 255);
border-style: outset;
border-width: 1px;
border-color: rgb(170, 0, 0);
</string>
              </property>
              <property name="text">
               <string> X</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Тип ткани</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Дата С:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="de_material_from">
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>По:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="de_material_to">
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="le_material_type">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton_10">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(170, 0, 0);
color: rgb(255, 255, 255);
border-style: outset;
border-width: 1px;
border-color: rgb(170, 0, 0);
</string>
              </property>
              <property name="text">
               <string> X</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Поставщик</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Рассчитать</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Сумма</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_material_sum">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <widget class="QTableWidget" name="tw_material">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>20</number>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>18</number>
          </attribute>
          <column>
           <property name="text">
            <string>Ткань</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Вес</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Приходов</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Сумма</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Фурнитура</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="1" column="0">
         <widget class="QTableWidget" name="tw_accessories">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>20</number>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>18</number>
          </attribute>
          <column>
           <property name="text">
            <string>Фурнитура</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Кол-во</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Приходов</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Сумма</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Тип фурнитуры</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Поставщик</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLineEdit" name="le_accessories_provider">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton_3">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton_12">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(170, 0, 0);
color: rgb(255, 255, 255);
border-style: outset;
border-width: 1px;
border-color: rgb(170, 0, 0);
</string>
              </property>
              <property name="text">
               <string> X</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLineEdit" name="le_accessories_type">
              <property name="dragEnabled">
               <bool>false</bool>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton_4">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton_13">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(170, 0, 0);
color: rgb(255, 255, 255);
border-style: outset;
border-width: 1px;
border-color: rgb(170, 0, 0);
</string>
              </property>
              <property name="text">
               <string> X</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Дата С:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="de_accessories_from">
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>По:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="de_accessories_to">
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Рассчитать</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Сумма</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_accessories_sum">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Прочее</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="1" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Дата С:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="de_comparing_from">
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>По:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="de_comparing_to">
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Затраты на :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cb_material_on">
              <property name="text">
               <string>Ткань</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cb_accessories_on">
              <property name="text">
               <string>Фурнитуру</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLineEdit" name="le_comparing_type">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton_5">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton_14">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(170, 0, 0);
color: rgb(255, 255, 255);
border-style: outset;
border-width: 1px;
border-color: rgb(170, 0, 0);
</string>
              </property>
              <property name="text">
               <string> X</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Тип затрат</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>Рассчитать</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QTableWidget" name="tw_comparing">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>20</number>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>18</number>
          </attribute>
          <column>
           <property name="text">
            <string>Затрата</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Кол-во</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Приходов</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Сумма</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Сумма</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_comparing_sum">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="pb_action"/>
  </widget>
  <action name="pb_action">
   <property name="text">
    <string>Печать</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>toolButton_10</sender>
   <signal>clicked()</signal>
   <receiver>le_material_type</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>525</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_11</sender>
   <signal>clicked()</signal>
   <receiver>le_material_provider</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_13</sender>
   <signal>clicked()</signal>
   <receiver>le_accessories_type</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>522</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_12</sender>
   <signal>clicked()</signal>
   <receiver>le_accessories_provider</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>521</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_14</sender>
   <signal>clicked()</signal>
   <receiver>le_comparing_type</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>525</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_view_material_name()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>448</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_view_provider_material()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_calc_material_supply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_view_accessories_name()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>497</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>-39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_view_provider_accessories()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>493</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>-59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_calc_accessories_supply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>604</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_5</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_view_comparing_name()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>498</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>496</x>
     <y>-72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_calc_comparing_supply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>-86</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ui_view_material_name()</slot>
  <slot>ui_view_provider_material()</slot>
  <slot>ui_calc_material_supply()</slot>
  <slot>ui_view_accessories_name()</slot>
  <slot>ui_view_provider_accessories()</slot>
  <slot>ui_calc_accessories_supply()</slot>
  <slot>ui_view_comparing_name()</slot>
  <slot>ui_calc_comparing_supply()</slot>
 </slots>
</ui>
